tools/ibc-test/Cargo.toml

```diff
@@ -45,7 +45,7 @@ toml_edit = "0.19.14"
 lazy_static = "1.4.0"
 ethers = { version = "2.0.2", features = ["rustls", "ws"] }
 
-forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", rev = "cd82be97" }
+forcerelay-ckb-sdk = { git = "https://github.com/synapseweb3/forcerelay-ckb-sdk", branch = "ckb-0.111" }
```

Comment:
This branch is already merged into main, though there is another breaking change in main (https://github.com/synapseweb3/forcerelay-ckb-sdk/pull/31).

Reply:
Yes, I will use the latest commit 64978b97